zero_bin/common/src/prover_state/persistence.rs

```diff
@@ -109,7 +109,7 @@ impl DiskResource for BaseProverResource {
             "{}/{}_base_{}_{}",
             &relative_circuit_dir_path(),
             PROVER_STATE_FILE_PREFIX,
-            env::var("EVM_ARITHMETIZATION_PKG_VER").unwrap_or("NA".to_string()),
+            env::var("EVM_ARITH_VER_KEY").unwrap_or("NA".to_string()),
```

Comment:
Why don't you use compile time `EVM_ARITHMETIZATION_PACKAGE_VERSION` instead of current process environment variable `EVM_ARITH_VER_KEY`, same like for `VERGEN_BUILD_TIMESTAMP`?

Reply:
EDIT: My mistake, this one should stay